src/Tickets/Commerce/Admin/Singular_Order_Notices.php

```diff
@@ -0,0 +1,240 @@
+<?php
+/**
+ * Singular order status notices.
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Admin
+ */
+
+namespace TEC\Tickets\Commerce\Admin;
+
+use Tribe__Admin__Notices;
+
+/**
+ * Class Singular_Order_Notices
+ *
+ * @since TBD
+ *
+ * @package TEC\Tickets\Commerce\Admin
+ */
+class Singular_Order_Notices {
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_STATUS_SUCCESSFULLY_UPDATED = 'updated';
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_STATUS_REFUNDED = 'status-refunded';
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_SUCCESSFULLY_REFUNDED = 'refunded';
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_PARTIALLY_REFUNDED = 'partial-refunded';
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_STATUS_CHANGE_FAILED = 'change-failed';
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_REFUND_FAILED = 'refund-failed';
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_GATEWAY_REFUND_FAILED = 'gateway-refund-failed';
+	/**
+	 * @since TBD
+	 *
+	 * @var string Notice key.
+	 */
+	const ORDER_GATEWAY_PARTIAL_REFUND_FAILED = 'gateway-partial-refund-failed';
+
+	/**
+	 * List of error notices for the admin.
+	 *
+	 * @since TBD
+	 *
+	 * @var string[]
+	 */
+	protected array $error_messages = [
+		self::ORDER_GATEWAY_PARTIAL_REFUND_FAILED,
+		self::ORDER_REFUND_FAILED,
+		self::ORDER_GATEWAY_REFUND_FAILED,
+		self::ORDER_STATUS_CHANGE_FAILED,
+	];
+
+	/**
+	 * List of successful admin notices.
+	 *
+	 * @since TBD
+	 *
+	 * @var string[]
+	 */
+	protected array $success_messages = [
+		self::ORDER_STATUS_REFUNDED,
+		self::ORDER_STATUS_SUCCESSFULLY_UPDATED,
+		self::ORDER_SUCCESSFULLY_REFUNDED,
+		self::ORDER_PARTIALLY_REFUNDED,
+	];
+
+	/**
+	 * This array contains the memoized translated order notices.
+	 *
+	 * @since TBD
+	 *
+	 * @var array<string,string> List of translated order notices.
+	 */
+	protected static array $generated_messages = [];
+
+	/**
+	 * @since TBD
+	 *
+	 * @return string[] Success message keys.
+	 */
+	public function get_success_keys(): array {
+		return $this->success_messages;
+	}
+
+	/**
+	 * @since TBD
+	 *
+	 * @return string[] Error message keys.
+	 */
+	public function get_error_keys(): array {
+		return $this->error_messages;
+	}
+
+	/**
+	 * Generates and returns the list of translated admin notices.
+	 *
+	 * @since TBD
+	 *
+	 * @return array<string, string> Key value of the translated admin notices.
+	 */
+	public function messages(): array {
+		if ( ! empty( self::$generated_messages ) ) {
+			return self::$generated_messages;
+		}
+
+		self::$generated_messages = [
+			/**
+			 * Success messages.
+			 */
+			/* translators: %1$s: order number. %2$s: status the order was changed from. %3$s: status the order was changed to. */
```

Comment:
For these comments, you don't need to include the extra characters in the placeholders.

```suggestion
			/* translators: 1. order number, 2. status the order was changed from, 3. status the order was changed to. */
```

Reply:
More context: https://developer.wordpress.org/apis/internationalization/internationalization-guidelines/#variables